src/components/Operations/CategoryIcon.jsx

```diff
@@ -0,0 +1,215 @@
+        return 'legalCounsel'
+      case '400330':
+        return 'homeAssistance'
+      case '400340':
+        return 'bankFees'
+      case '400350':
+        return 'financialAdvisor'
+      case '400400':
+        return 'kids'
+      case '400410':
+        return 'kidsAllowance'
+      case '400420':
+        return 'schoolRestaurant'
+      case '400430':
+        return 'childCare'
+      case '400440':
+        return 'schoolInsurance'
+      case '400450':
+        return 'toysAndGifts'
+      case '400460':
+        return 'pensionPaid'
+      case '400470':
+        return 'kidsActivities'
+      case '400500':
+        return 'tax'
+      case '400510':
+        return 'incomeTax'
+      case '400520':
+        return 'socialTax'
+      case '400530':
+        return 'wealthTax'
+      case '400540':
+        return 'realEstateTax'
+      case '400600':
+        return 'health'
+      case '400610':
+        return 'healthExpenses'
+      case '400620':
+        return 'healthInsurance'
+      case '400700':
+        return 'activities'
+      case '400710':
+        return 'activityFees'
+      case '400720':
+        return 'activityEquipments'
+      case '400730':
+        return 'activityLessons'
+      case '400740':
+        return 'electronicsAndMultimedia'
+      case '400750':
+        return 'booksMoviesMusic'
+      case '400760':
+        return 'hobbyAndPassion'
+      case '400800':
+        return 'goingOutAndTravel'
+      case '400810':
+        return 'restaurantsAndBars'
+      case '400820':
+        return 'goingOutEntertainment'
+      case '400830':
+        return 'goingOutCulture'
+      case '400840':
+        return 'travel'
+      case '400850':
+        return 'journey'
+      case '400900':
+        return 'educationAndTraining'
+      case '400910':
+        return 'tuition'
+      case '400920':
+        return 'eduBooksAndSupplies'
+      case '400930':
+        return 'studentLoan'
+      case '400940':
+        return 'eduLessons'
+      case '401000':
+        return 'homeAndRealEstate'
+      case '401010':
+        return 'realEstateLoan'
+      case '401020':
+        return 'rent'
+      case '401030':
+        return 'homeCharges'
+      case '401040':
+        return 'homeInsurance'
+      case '401050':
+        return 'homeImprovement'
+      case '401060':
+        return 'homeHardware'
+      case '401070':
+        return 'water'
+      case '401080':
+        return 'power'
+      case '600000':
+        return 'excludeFromBudgetTheme'
+      case '600100':
+        return 'excludeFromBudgetCat'
+      case '600110':
+        return 'internalTransfer'
+      case '600120':
+        return 'creditCardPayment'
+      case '600130':
+        return 'loanCredit'
+      case '600140':
+        return 'professionalExpenses'
+      case '600150':
+        return 'investmentBuySell'
+      case '600160':
+        return 'friendBorrowing'
+      case '600170':
+        return 'savings'
```

Comment:
It would have been even better to directly use cozy-ui icons: https://docs.cozy.io/cozy-ui/react/#/Basics?id=icon
But, I'm not sure all banks icons are defined, and it would take time to migrate, so let's do it like this!